packages/chat-headless/src/ChatHeadless.ts

```diff
@@ -136,6 +125,32 @@ export class ChatHeadless {
     };
   }
 
+  /**
+   * Loads the {@link ConversationState} from session storage, if present,
+   * and adds a listener to keep the conversation state in sync with the stored
+   * state
+   *
+   * @remarks
+   * This is called by default if {@link HeadlessConfig.saveToSessionStorage} is
+   * true.
+   *
+   * @public
+   */
+  initSessionStorage() {
```

Comment:
to double check, did you run `npm run test` to ensure the existing tests still pass?